skeleton/docker-entrypoint.sh

```diff
@@ -97,7 +97,7 @@ fi
 $sudo $VENVBIN/python /app/scripts/cors.py
 
 # Handle ADDONS installation
-if [[ -v ADDONS ]]; then
+if [[ -n "${ADDONS}" ]]; then
```

Comment:
I think the -n fix should be done everywhere.  The code I've committed I think already does this for at least one var.  It is customary to treat empty variables exactly as absent, and that is why -n is useful.